src/components/Calculator.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+
+const Calculator = () => {
```

Comment:
> Create at least 1 child component
- [ ] As stated in the project requirement, calculator.js  is meant to have at least 1 child functional component, I'll suggest you create anew child component and import it into `Calculator.js`. You can consider extracting the calculator screen into its own component. :smile: :small_airplane: